frigg/deployments/models.py

```diff
@@ -52,10 +52,16 @@ def is_pending(self):
     @property
     def queue_object(self):
         obj = self.build.queue_object
+
+        if self.image and self.image != "":
```

Comment:
Is this change necessary? As self.image is already [set correctly](https://github.com/frigg/frigg-hq/pull/250/files#diff-01d4d2de9b7ba8519ce768d37eae5c81R278).